papermill/cli.py

```diff
@@ -10,10 +10,18 @@
 
 from .execute import execute_notebook
 from .iorw import read_yaml_file
+from . import __version__ as papermill_version
 
 click.disable_unicode_literals_warning = True
 
 
+def print_the_paermill_version(ctx, param, value):
+    if not value:
+        return
+    print(papermill_version)
```

Comment:
Let's make this slightly more verbose (like how pip --version works)

> $ pip --version
> pip 18.0 from /home/mseal/.py2local/local/lib/python2.7/site-packages/pip (python 2.7)

```
    print("{version} from {path} ({pyver})".format(
          version=papermill_version,
          path=__file__,
          pyver=platform.python_version()))
```
should do the trick

Reply:
@MSeal Done!!